server/controllers/work/deleteWork.js

```diff
@@ -0,0 +1,28 @@
+const { Work } = require('../../database/models');
+const { customError } = require('../errors');
+const { paramsValidation } = require('../../utils/validation');
+
+const deleteWork = async (req, res, next) => {
+  try {
+    const { userId } = req.body;
```

Comment:
actually work has a userID as `userId`,
So we'll get userid from it not from request body